policykit/policyengine/utils.py

```diff
@@ -46,4 +48,18 @@ def construct_authorize_install_url(request, integration, community=None):
     community_slug = community.metagov_slug if community else ''
     url = f"{settings.METAGOV_URL}/auth/{integration}/authorize?type=app&community={community_slug}&redirect_uri={encoded_redirect_uri}&state={state}"
     logger.debug(url)
-    return url
\ No newline at end of file
+    return url
+
+def get_starterkits_info():
+    """
+    Get a list of all starter-kit names and descriptions.
+    """
+    starterkits = []
+    for kit_file in os.listdir('./starterkits'):
+        f = open(f'{os.getcwd()}/starterkits/{kit_file}')
```

Comment:
Has this been tested on a server that's serving up files using apache web server? Just want to check, since the `cwd` command there might differ from what you get locally.

Reply:
Tested and updated!